beacon-chain/sync/service.go

```diff
@@ -205,34 +208,35 @@ func (ss *Service) receiveBlock(msg p2p.Message) {
 		return
 	}
 
-	// Verify attestation coming from proposer then forward block to the subscribers.
-	attestation := types.NewAttestation(response.Attestation)
-	cState := ss.chainService.CurrentCrystallizedState()
-	parentSlot, err := ss.chainService.BlockSlotNumberByHash(block.ParentHash())
-	if err != nil {
-		log.Errorf("Failed to get parent slot: %v", err)
-		return
-	}
-	proposerShardID, _, err := casper.ProposerShardAndIndex(cState.ShardAndCommitteesForSlots(), cState.LastStateRecalc(), parentSlot)
-	if err != nil {
-		log.Errorf("Failed to get proposer shard ID: %v", err)
-		return
-	}
-	// TODO(#258): stubbing public key with empty 32 bytes.
-	if err := attestation.VerifyProposerAttestation([32]byte{}, proposerShardID); err != nil {
-		log.Errorf("Failed to verify proposer attestation: %v", err)
-		return
+	if ss.enableAttestationValidity {
```

Comment:
I'm not sure why this is disabled for the demo. All that's needed here is an attestation with the first bit in the bitfield set to 1. Is that not possible with the current setup?